ui/src/views/Hooks/ViewHook/index.jsx

```diff
@@ -51,14 +45,13 @@ export default class ViewHook extends Component {
           hookGroupId,
           payload,
         },
-        refetchQueries: ['Hook'],
-        awaitRefetchQueries: true,
       });
 
       this.props.history.push(
         `/hooks/${encodeURIComponent(hookGroupId)}/${hookId}`
       );
 
+      await this.props.data.refetch();
```

Comment:
I think `refetchQueries` and `awaitRefetchQueries` does the same thing. Any reason for that?